src/config/environment.rs

```diff
@@ -22,53 +22,61 @@ lazy_static! {
         if *SIMULATOR_ENABLED { 3 } else { 2 };
 }
 
+pub const FACTORF64: f64 = 2.0;
```

Comment:
Maybe we can get this value from an env variable at compile-time (see [option_env!](https://doc.rust-lang.org/std/macro.option_env.html)), and use `1` as default value?

Something like this? (not tested)

```suggestion
pub const FACTOR: f64 = option_env!("SCALE_FACTOR").and_then(|f| f64::from_str(&f).ok()).unwrap_or(1.0);
```

If necessary this can also be the place to validate the value (and if not valid we take the default one).

Reply:
Unfortunately, const types doesnt support non const operation.